packages/storybook/stories/Theming.stories.mdx

```diff
@@ -1,60 +1,50 @@
+import DefaultThemeSnippetText from '!!raw-loader!./snippets/DefaultTheme.snippet.tsx';
 import ThemedButtonText from '!!raw-loader!./snippets/ThemedButton.snippet.tsx';
 
 <Meta id="theming" title="Concepts/Theming" />
 
 # Theming
 
-ACS UI SDK use components and icons from both [Fluent UI](https://developer.microsoft.com/fluentui#/controls/web)
+ACS UI Library uses components and icons from both [Fluent UI](https://aka.ms/fluentui#/controls/web)
 and [Fluent React Northstar](https://fluentsite.z22.web.core.windows.net). The `FluentThemeProvider`
-component ensures:
-
-1. ACS UI SDK and Fluent UI components and icons are themed together
-2. Fluent React Northstar icons are provided
+is a wrapping component that allows you to theme all ACS UI Library components consistently.
 
 ## Usage
 
-To use the FluentThemeProvider, simply import it from `@azure/communication-react` and wrap it around
-ACS UI SDK components and Fluent UI components. For the example below we will provide theming and icons to
-the `ControlBar`.
+ACS UI Library components are themed with a light theme by default as shown by the `ControlBar` and buttons below.
 
-<Source code={FluentThemeProviderSnippetText} />
+<Source code={DefaultThemeSnippetText} />
 
 <Canvas withSource="none">
-  <FluentThemeProviderSnippet />
+  <DefaultThemeSnippet />
 </Canvas>
 
-`FluentThemeProvider` does two things here:
-
-1. `FluentThemeProvider` loads the icons used by components such as `CameraButton`
-2. `FluentThemeProvider` applies theme (light theme by default) to the `ControlBar`
-
-To illustrate how another theme can be applied, import `darkTheme` from `@azure/communication-react`
-and assign it to the `fluentTheme` property.
+To theme this ControlBar, import `FluentThemeProvider` from `@azure/communication-react` and wrap it around the `ControlBar`.
+Then import `darkTheme` from `@azure/communication-react` and assign it to the `fluentTheme` property of `FluentThemeProvider`.
+The import `darkTheme` is a [Theme](https://aka.ms/fluentui#/controls/web/references/theme) object from
+Fluent UI.
 
 <Source code={DarkControlBarText} />
 
 <Canvas withSource="none">
   <DarkControlBar />
 </Canvas>
 
-The dark theme is applied to the `ControlBar`. The icon colors in the predefined button components are themed correctly
-because they are built using the `DefaultButton` Fluent UI component.
-
 ## FluentThemeProvider Props
 
 <Props of={FluentThemeProvider} />
 
 ## Creating your own theme
 
-If you wish to use your own theme, we strongly advise using Microsoft's [Fluent UI Theme Designer](https://aka.ms/themedesigner).
-Please ensure that the Accessibility checker on the same page passes when creating your own theme.
+The imports `lightTheme` and `darkTheme` are available from `@azure/communication-react`. But if you wish to use your own theme,
+we strongly advise using Microsoft's [Fluent UI Theme Designer](https://aka.ms/themedesigner) to create your theme. Please ensure
```

Comment:
nit: strongly advise -> recommend

I think it's worth being polite in these docs.